src/main/java/seedu/address/logic/parser/AddCommandParser.java

```diff
@@ -116,6 +117,23 @@ public AddCommand parse(String args) throws ParseException {
             Skill skill = new Skill(name, level, tagList);
             return new AddSkillCommand(skill);
 
+        case(ItemUtil.NOTE_ALIAS):
```

Comment:
No action needs to be taken here, but we need to standardise the style in the future as per #83